frontend/packages/ceph-storage-plugin/src/components/block-pool/block-pool-list-page.tsx

```diff
@@ -146,7 +150,10 @@ const BlockPoolTableRow: React.FC<RowFunctionArgs> = ({ obj, index, key, style,
         className={blockPoolColumnInfo.name.classes}
         columnID={blockPoolColumnInfo.name.id}
       >
-        <ResourceLink kind={referenceFor(obj)} name={name} namespace={namespace} />
+        <ResourceIcon kind={referenceForModel(CephBlockPoolModel)} />
+        <Link to={to} className="co-resource-item__resource-name" data-test-operand-link={name}>
```

Comment:
```suggestion
        <Link to={to} className="co-resource-item__resource-name" data-test={name}>
```
This is not an operand link. We are not in the operators page.

Reply:
ack